operators/db2/secret/ibm-entitled-registry-credentials-secret.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: ibm-entitled-registry-credentials
```

Comment:
Careful - by default the name of the created secret follows the name of the "parent" sealed secret. This means the name of the secret will be "ibm-entitled-registry-credentials" which is more than fine by the way.

The only slight issue is that in a lot of the templates used, "ibm-entitlement-key" is used instead of "ibm-entitled-registry-credentials" (For instance here: https://github.com/apollo-business-automation/ibm-cp4ba-enterprise-deployment/blob/main/scripts/roles/db2/templates/db2ucluster.yaml.j2) 

I reckon it would be a good idea to change the name of this to "ibm-entitlement-key" to ensure consistency in this approach.